Add documentation for new Identity Token `.well-known` endpoint used for Plugin WIF (#25469)
hashicorp/vault: website/content/api-docs/secret/identity/tokens.mdx

```json
{

>   "issuer": "https://example.com/v1/identity/oidc",

Reviewer: Thanks!

  "authorization_endpoint": "",
  "token_endpoint": "",
  "jwks_uri": "https://example.com/v1/identity/oidc/.well-known/keys",
  "response_types_supported": null,
  "subject_types_supported": ["public"],
  "id_token_signing_alg_values_supported": ["RS256"],
  "scopes_supported": null,
  "token_endpoint_auth_methods_supported": null,
  "claims_supported": null
}
```

## Read plugin identity .well-known configurations

> Query this path to retrieve a set of claims about the plugin identity tokens' configuration. The response is a compliant [OpenID Provider Configuration Response](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationResponse).

Reviewer 2: ```suggestion
Use the token endpoint to retrieve an
[OpenID Provider Configuration Response](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationResponse)
with claims about the identity token configuration for a plugin.
```

Style correction: write in active voice

Author: Updated, thanks!

| Method | Path                                             |
| :----- | :----------------------------------------------- |
| `GET`  | `identity/oidc/plugins/.well-known/openid-configuration` |

### Sample request

```shell-session
$ curl \
    --request GET \
    http://127.0.0.1:8200/v1/identity/oidc/plugins/.well-known/openid-configuration
```

### Sample response

```json
{

>   "issuer": "https://example.com/v1/identity/oidc",

Reviewer: ```suggestion
  "issuer": "https://example.com/v1/identity/oidc/plugins",
```
